.travis.yml

```diff
@@ -1,14 +1,14 @@
+sudo: required
```

Comment:
[…]
> Of course you can also see this "variability" as a good thing since it can help you detect bugs and track compatibility but given the type of JRuby support you describe, you might actually want to lock the version and leave it up to actual JRuby users to keep it up to date via contributions.
[…]

That is the intent, CI in this case might help us detect bugs on
jruby, and if those are trivials we will fix them.

I'm sure we would be happy to lock the version (or other changes) if
any contributors wish to help for jruby support, provided they use
jruby actively of course.